### ISSUE TITLE
Divide line after last tab is thinner

### ISSUE THREAD

**Issue by BPower0036** [black7375/Firefox-UI-Fix#104]
**Describe the bug**
The divide line after the last tab is thinner then the others.

**Expected behavior**
It should be the same width

**Screenshots**
![afbeelding](https://user-images.githubusercontent.com/80090789/123038573-f753ad00-d3f0-11eb-9ad1-0ea9213b2457.png)

**Environment (please complete the following information):**
Check like `- [x]`.
 - Distribution
   - [x] [Original Lepton](https://github.com/black7375/Firefox-UI-Fix)
   - [ ] [Lepton's photon style](https://github.com/black7375/Firefox-UI-Fix/tree/photon-style)
 - Firefox Version: [89.0.1]
 - OS:
   - [ ] Linux
   - [x] Windows
   - [ ] Mac
   - [ ] Others: 
 - Theme:
   - [ ] Light
   - [ ] Dark
   - [ ] Alpenglow(System: Light)
   - [ ] Alpenglow(System: Dark)
   - [x] System Default
     - GTK Theme Name(Only Linux User): [sample] Adwaita, Adwaita-Dark, Breath, Breath-Dark

**Additional context**

**Comment by black7375** [comment]
Is it Windows7?

**Comment by BPower0036** [comment]
No it is Windows 10

**Comment by black7375** [comment]
It seems a CSS decimal point issue.

My computer did this and the symptoms went away.
https://github.com/black7375/Firefox-UI-Fix/blob/9f71fd285b71e375b62b1b8306670305d116b897/userChrome.css#L323

to
```css
 transform: translateX(1.5px) translateY(calc(-50% + 1px)) !important; 
```
- `0` => Normal
- `.5px` => Thin
- `1px` => Thin
- `1.5px` => Normal

---

Before
![image](https://user-images.githubusercontent.com/25581533/123277322-88ac4780-d4f5-11eb-843d-29aefcdd02b0.png)

After
![image](https://user-images.githubusercontent.com/25581533/123277112-569ae580-d4f5-11eb-960f-7049f03e1fc2.png)

Please test it.
If it works on your computer, I'll apply it right away.

**Comment by BPower0036** [comment]
Yes, this solved the problem.
Thanks
